common/persistence/nosql/nosqlplugin/cassandra/plugin.go

```diff
@@ -96,9 +113,9 @@ func toGoCqlConfig(cfg *config.NoSQL) gocql.ClusterConfig {
 		MaxConns:              cfg.MaxConns,
 		TLS:                   cfg.TLS,
 		ProtoVersion:          cfg.ProtoVersion,
-		Consistency:           cassandraDefaultConsLevel,
-		SerialConsistency:     cassandraDefaultSerialConsLevel,
-		Timeout:               defaultSessionTimeout,
-		ConnectTimeout:        defaultConnectTimeout,
+		Consistency:           gocql.ParseConsistency(cfg.Consistency),
+		SerialConsistency:     gocql.ParseSerialConsistency(cfg.SerialConsistency),
```

Comment:
can you change these functions to return err instead of panicing? I know `toGoCqlConfig` is not returning error but it's only caller is `doCreateDB` which returns error so we can propagate these errors.

Reply:
PR for this change: https://github.com/uber/cadence/pull/5811